source/references.rst

```diff
@@ -14,3 +14,5 @@
 .. [TFAFFAMB] `TF-A Secure Partition Manager: FF-A manifest binding to device tree <https://trustedfirmware-a.readthedocs.io/en/latest/components/ffa-manifest-binding.html>`
 
 .. [TCG_EFI] `https://trustedcomputinggroup.org/resource/tcg-efi-protocol-specification`.
+
+.. [TF_BL31] `Data structures used in the BL31 cold boot interface <https://trustedfirmware-a.readthedocs.io/en/latest/design/firmware-design.html#data-structures-used-in-the-bl31-cold-boot-interface>`
```

Comment:
Sorry, just spotted that the hyperlink syntax in these references are wrong (needs a trailing __). Can you fix this? See https://github.com/FirmwareHandoff/firmware_handoff/pull/39 that fixes the other links.